src/main/scala/uk/gov/nationalarchives/tdr/api/http/RouteLogging.scala

```diff
@@ -0,0 +1,32 @@
+package uk.gov.nationalarchives.tdr.api.http
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.server.RouteResult.{Complete, Rejected}
+import akka.http.scaladsl.server.directives.{DebuggingDirectives, LoggingMagnet}
+import akka.http.scaladsl.server.{Directive0, RouteResult}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.Materializer
+import com.typesafe.scalalogging.Logger
+
+object RouteLogging {
+  implicit val actorSystem: ActorSystem = ActorSystem("graphql-server")
```

Comment:
Is this line necessary? I wouldn't have expected this object to be using a completely different actor system to Routes.

Reply:
Removed logging as an abstracted object